theme/sass/components/_footer.scss

```diff
@@ -1,31 +1,50 @@
 .footer {
     display: flex;
     justify-content: center;
-    background-color: $accent-color-two;
+    background-color: $secondary-color;
+    height: 82%;
+
+    @include breakpoint($md) {
+        height: auto;
+    }
 
     .footer-container {
         display: flex;
         margin: 0;
         width: $gutter-size;
-        align-items: flex-start;
+        flex-direction: column;
+        
+        @include breakpoint($md) {
+            align-items: flex-start;
+            flex-direction: row;
+        }
 
         .copyright-statement {
-            flex: 0 0 20%;
-            max-width: 20%;
             color: white;
             font-size: 12px;
+            position: absolute;
+            bottom: 10px;
```

Comment:
Doing, position absolute and forcing the item to go down with bottom isn't the best way to do this, check out flex order property. That will do what you want in a much better way. https://css-tricks.com/snippets/css/a-guide-to-flexbox/